aws/cloudwatch_alarm.go

```diff
@@ -122,25 +122,26 @@ func nukeAllCloudWatchAlarms(session *session.Session, identifiers []*string) er
 		}
 	}
 
-	input := cloudwatch.DeleteAlarmsInput{AlarmNames: identifiers}
-	_, err = svc.DeleteAlarms(&input)
-
-	// Record status of this resource
-	e := report.BatchEntry{
-		Identifiers:  aws.StringValueSlice(identifiers),
-		ResourceType: "CloudWatch Alarm",
-		Error:        err,
-	}
-	report.RecordBatch(e)
+	for _, alarm := range identifiers {
+		_, err = svc.DeleteAlarms(&cloudwatch.DeleteAlarmsInput{AlarmNames: []*string{alarm}})
+
+		// Record status of this resource
+		e := report.BatchEntry{
```

Comment:
This will report the success/failure of the entire batch based on an individual record. Since non-composite alarms can be deleted multiple at a time I would propose reverting this change and adding logic above where we loop through the composites for deleting the composite alarm, then removing the composite alarm from the list of identifiers to delete in this `svc.DeleteAlarms` call